### PR TITLE
post on first Lean release

### PR DESCRIPTION
**Pull request by semorrison**

### CODE REVIEW

File: posts/first-lean-release.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+rather than fixed versions of upstream dependencies.
+
+As Lean matures into a significant programming language,
+as well as an even better theorem prover for modern mathematics and software/hardware verification,
+the ecosystem will grow.
+We want to make sure that everyone can find compatible versions of libraries,
+and that everything **just works**.
+
+It's a long road to get there,
+but the Lean Focused Research Organization intends to dedicate resources
```
Comment (eric-wieser) [inline posts/first-lean-release.md:31]: A link to leanFRO's website would be good here

Comment (semorrison) [reply posts/first-lean-release.md]: Done!

File: posts/first-lean-release.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+author: 'Scott Morrison'
+category: 'announcement'
+date: 2023-08-29 16:30:00 UTC+02:00
+description: 'The first official release of Lean 4, and our plan for regular releases.'
```
Comment (eric-wieser) [inline posts/first-lean-release.md:5]: Is "our" correct here? Is this a lean-community plan or a lean FRO plan?

File: posts/first-lean-release.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+author: 'Scott Morrison'
+category: 'announcement'
+date: 2023-08-29 16:30:00 UTC+02:00
```
Comment (eric-wieser) [inline posts/first-lean-release.md:4]: Are you now in europe? Or was this written in Leiden?

**Review by eric-wieser — APPROVED** [review]

File: posts/first-lean-release.md (hunk 2 above)
Comment (semorrison) [reply posts/first-lean-release.md:5]:
```suggestion
description: 'The first official release of Lean 4, and plans for regular releases.'
```

File: posts/first-lean-release.md (hunk 3 above)
Comment (semorrison) [reply posts/first-lean-release.md:4]:
```suggestion
date: 2023-09-08 15:58:40 UTC+10:00
```

**Pull request merged**